blindbid.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+package rusk;
+
+import "basic_fields.proto";
+import "transaction.proto";
+import "keys.proto";
+
+message GenerateScoreRequest {
```

Comment:
`secret_k` and `tree_root` should be also provided

Reply:
About bid_tree_root is obtained in rusk's side. Since it has the storage, it can check the root of the tree. Go has no way to know it. Therefore, it should not be added in the request.

Secret_k makes sense, I forgot. Will add it